src/components/TableHeadingCellEnhancer.js

```diff
@@ -1,21 +1,4 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import compose from 'recompose/compose';
-import mapProps from 'recompose/mapProps';
-import getContext from 'recompose/getContext';
-import { combineHandlers } from '../utils/compositionUtils';
-
-const EnhancedHeadingCell = OriginalComponent => compose(
-  getContext({
-    events: PropTypes.object,
-  }),
-  mapProps(({ events: { onSort }, ...props }) => ({
-    ...props,
-    onClick: combineHandlers([
-      () => onSort && onSort(props.sortProperty || { id: props.columnId }),
-      props.onClick,
-    ]),
-  }))
-)(props => <OriginalComponent {...props} />);
+// Obsolete
+const EnhancedHeadingCell = OriginalComponent => OriginalComponent;
```

Comment:
Can this be removed?

Reply:
It can, but it's _technically_ a breaking change. Glad to remove if you're not concerned.

Reply:
Just to double confirm - we're leaving this here to avoid causing a potential breaking change right? I think we're maybe just outside the realm of where a 2.0 release makes sense. If I'm understanding correctly, we can leave this here for now to avoid breaking change and remove the obsolete field in a 2.0 version.

Thoughts?

Reply:
Correct. I left it as a no-op to be cleaned up for a 2.0 release.

The chances someone is actually depending on this seem quite low, but it's been exported as part of the public API.